index.html

```diff
@@ -139,7 +139,7 @@ <h2>Development</h2>
 
 <h2>Input into the process</h2>
 
-<p>Ideas for evolving the ECMAScript language are accepted in any form. Any discussion, idea or proposal for a change or addition which has not been submitted as a formal proposal is considered to be a “strawman” (stage 0) and has no acceptance requirements. Such submissions must either come from members of TC39 or from non-members who have <a href="http://www.ecma-international.org/memento/register_TC39_Royalty_Free_Task_Group.php">registered</a> via Ecma International.
+<p>Ideas for evolving the ECMAScript language are accepted in any form. Any discussion, idea or proposal for a change or addition which has not been submitted as a formal proposal is considered to be a “strawman” (stage 0) and has no acceptance requirements. Such submissions must either come from members of TC39 or from non-members who have <a href="http://www.ecma-international.org/memento/join.htm">registered</a> via Ecma International.
```

Comment:
I think contextually speaking this is a better link to use: https://tc39.github.io/agreements/contributor/

Reply:
Awesome! I've rebased to include the link you've suggested. I also updated the first link to `https`.